jac-uk/admin
# #1666 Stages and statuses

## What's included?
Introduces our new set of Stages and Statuses for applications.

Main ticket: #1666 

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
Create or use an existing exercise which has applications.
Begin processing applications and navigate to Stages.

- Check that applications can be assigned the statuses and can progress through the amended stages
- Check that page size can be amended
- Check that search works
- Check that filter works

## Risk - how likely is this to impact other areas?
🟠 Medium risk - this does change code that is shared with other areas

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
